test/rabbit_watchdog_management_SUITE.erl

```diff
@@ -0,0 +1,122 @@
+%% The contents of this file are subject to the Mozilla Public License
+%% Version 1.1 (the "License"); you may not use this file except in
+%% compliance with the License. You may obtain a copy of the License
+%% at http://www.mozilla.org/MPL/
+%%
+%% Software distributed under the License is distributed on an "AS IS"
+%% basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See
+%% the License for the specific language governing rights and
+%% limitations under the License.
+%%
+%% The Original Code is RabbitMQ Watchdog.
+%%
+%% The Developer of this component is Erlang Solutions, Ltd.
+%% Copyright (c) 2017-2018 Erlang Solutions, Ltd. All rights reserved.
+%%
+
+-module(rabbit_watchdog_management_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("rabbit_watchdog.hrl").
+
+-define(COLLECT_INTERVAL, 100).
+-define(WATCHDOG_INTERVAL, 200).
+-define(RABBITMQ_MGMT_STARTUP_DELAY, 5000).
```

Comment:
This is a magic number, probably would be better to do some kind of retry logic with a timeout. In case the plugin would start a lot earlier.

Reply:
Done.